src/fu-tool.c

```diff
@@ -63,6 +63,15 @@ fu_sort_command_name_cb (FuUtilItem **item1, FuUtilItem **item2)
 	return g_strcmp0 ((*item1)->name, (*item2)->name);
 }
 
+static void
+fu_util_test_for_sandbox (const gchar *value)
```

Comment:
couldn't this just take the GError as a parameter and use `g_prefix_error()`? It seems a shame to just put it in a `g_warning`... Ack on the first two patches tho.

Reply:
fixed